sdk/examples/churn_prediction/Build and Deploy Churn Ensemble.ipynb

```diff
@@ -1499,6 +1499,43 @@
     "avg_pred_churn_metric.get()"
```

Comment:
<p>Can we actually execute this command show the output? That way a user who's just skimming the notebook or reading the docs can get a sense of what preset metrics we have.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aqueducthq/aqueduct/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>